frontend/packages/client/src/hooks/useCommunity.js

```diff
@@ -106,17 +112,20 @@ export default function useCommunity() {
             creatorAddr,
             additionalAuthors: listAddrAuthors?.map((ele) => ele.addr),
             additionalAdmins: listAddrAdmins?.map((ele) => ele.addr),
-            proposalThreshold,
             slug,
             githubUrl,
             instagramUrl,
             twitterUrl,
             websiteUrl,
             discordUrl,
             logo: communityLogo?.fileUrl,
-            contractAddress,
-            contractName,
-            storagePath,
+            contractAddress: setDefaultValue(
+              contractAddr,
+              '0x0ae53cb6e3f42a79'
+            ),
+            contractName: setDefaultValue(contractN, 'FlowToken'),
+            storagePath: setDefaultValue(storageP, 'flowTokenBalance'),
+            proposalThreshold: setDefaultValue(propThreshold, '0'),
```

Comment:
as discussed with @rvmelkonian if this fields are not filled in the from, frontend needs to send these values as default